internal/provider/resources/block.go

```diff
@@ -27,9 +29,9 @@ type BlockResourceModel struct {
 	AccountID   customtypes.UUIDValue      `tfsdk:"account_id"`
 	WorkspaceID customtypes.UUIDValue      `tfsdk:"workspace_id"`
 
-	Name types.String         `tfsdk:"name"`
-	Type types.String         `tfsdk:"type"`
-	Data jsontypes.Normalized `tfsdk:"data"`
+	Name     types.String         `tfsdk:"name"`
+	TypeSlug types.String         `tfsdk:"type_slug"`
```

Comment:
Good change, ran across this when working on `Update` and wasn't sure what `Type` referred to here until looking at the schema description.